File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 25 e 26(Manzano)/js/exercicioE.js
/*e) Efetuar o cálculo e a apresentação do valor de uma prestação em atraso, utilizando a fórmula
PRESTACAO <- VALOR + (VALOR * TAXA/100) * TEMPO). */

var valor = parseInt(prompt("Digite o valor da prestação:"))
var tempoAtraso = parseInt(prompt("Digite o tempo em meses do atraso da prestação:"))
var taxa = parseInt(prompt("Digite o valor em porcentagem da taxa de juros prestação:"))
var prestacao = valor+(valor*taxa/100)* tempoAtraso
alert("O valor das prestações atrasadas com os juros é: "+prestacao)

File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag25e26/ExercicioE.java
package manzanoPag25e26;

import java.util.Scanner;

/*e) Efetuar o cálculo e a apresentação do valor de uma prestação em atraso, utilizando a fórmula
PRESTACAO <- VALOR + (VALOR * TAXA/100) * TEMPO). */

public class ExercicioE {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int valor, tempoAtraso, taxa;
		float prestacao;

		System.out.println("Digite o valor da prestação:");
		valor = sc.nextInt();
		System.out.println("Digite o tempo em meses do atraso da prestação:");
		tempoAtraso = sc.nextInt();
		System.out.println("Digite o valor em porcentagem da taxa de juros prestação:");
		taxa = sc.nextInt();
		
		prestacao = valor + (valor * taxa / 100) * tempoAtraso;

		System.out.println("O valor das prestações atrasadas com os juros é: " + prestacao);

		sc.close();

	}

}
